src/MSALWrapper/AuthFlow/IntegratedWindowsAuthentication.cs

```diff
@@ -67,7 +67,6 @@ protected override async Task<TokenResult> GetTokenInnerAsync()
                 && ex.Message.StartsWith("AADSTS50076", StringComparison.OrdinalIgnoreCase))
             {
                 this.logger.LogWarning("Warning: IWA failed, 2FA is required.");

```

Comment:
Should we degrade this to Debug? or remove the 2FA part as well?

Reply:
The warning message is technically still accurate, in that the MSAL exception does complain saying that an MFA challenge is required. If anything, I'd want to perhaps add some more detail to avoid confusion around people expecting an MFA prompt.

I'm also open to the idea of changing the verbosity level of this message. What's your preference?

Reply:
I would vote for changing the verbosity level as people are less concerned about IWA failing especially with the recent changes.

Reply:
My only concern with 2FA required message is if people are receiving a challenge (Authenticator app/Something else). If not, switching it to debug seems like a safer option.

Reply:
From my understanding, IWA will never trigger a 2FA challenge. It's simply stating that one is required, but since IWA is a purely silent flow, it will choose to silently fail instead of prompting the user.

Reply:
ah, that's correct. Well in that case 2FA required does seem a bit inactionable so lowering it to debug makes sense to me.

Reply:
done!